docs/excel/excel-add-ins-comments.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Work with comments using the Excel JavaScript API (preview)
+description: ''
+ms.date: 10/15/2019
+localization_priority: Normal
+---
+
+# Work with comments using the Excel JavaScript API (preview)
+
+> [!NOTE]
+> The comment APIs are currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+This article describes how to add, read, modify, and remove comments in a workbook with the Excel JavaScript API. You can learn more about the comment feature from the [Insert comments and notes in Excel](https://support.office.com/article/insert-comments-and-notes-in-excel-bdcc9f5d-38e2-45b4-9a92-0b2b5c7bf6f8) article.
+
+In the Excel JavaScript API, a comment is both the initial note and the connected threaded discussion. It is tied to an individual cell. Every comment can be replied to by people viewing the workbook (that have sufficient permissions). A [Comment](/javascript/api/excel/excel.comment) object stores those replies as [CommentReply](/javascript/api/excel/excel.commentreply) objects. You should consider a comment to be a thread and that a thread must have a special entry as the starting point.
+
+![An Excel comment, labelled "Comment" with two replies, labelled "Comment.replies[0]" and "Comment.replies[1].](../images/excel-comments.png)
+
+Comments within a workbook are tracked by the `Workbook.comments` property. This includes comments created by users and also comments created by your add-in. The `Workbook.comments` property is a [CommentCollection](/javascript/api/excel/excel.commentcollection) object that contains a collection of [Comment](/javascript/api/excel/excel.comment) objects. Comments are also accessible at the [Worksheet](/javascript/api/excel/excel.worksheet) level. The samples in this article work with comments at the workbook level, but they can be easily modified to use the `Worksheet.comments` property.
+
+## Add comments
+
+To add comments to a workbook, use the `CommentCollection.add` method, passing in the cell where the comment will be added, as either a string or [Range](/javascript/api/excel/excel.range) object, and the comment's text, as a string. Comments added by an add-in are attributed to the current user of that add-in.
```

Comment:
Optionally could make this bullets, e.g. 
Use the method, like so:
-Pass in the cell 
- Add comment text as string